build.sbt

```diff
@@ -260,10 +261,9 @@ lazy val tests = project
         // See https://github.com/sbt/sbt/issues/3618#issuecomment-448951808
         "javax.ws.rs" % "javax.ws.rs-api" % "2.1.1" artifacts Artifact("javax.ws.rs-api", "jar", "jar"),
         "org.testcontainers" % "kafka" % testcontainersVersion % Test,
-        "org.apache.commons" % "commons-compress" % "1.18", // embedded Kafka pulls in Avro, which pulls in commons-compress 1.8.1, see testing.md
         "org.scalatest" %% "scalatest" % scalatestVersion % Test,
         "io.spray" %% "spray-json" % "1.3.5" % Test,
-        "com.fasterxml.jackson.core" % "jackson-databind" % "2.9.8" % Test, // ApacheV2
+        "com.fasterxml.jackson.core" % "jackson-databind" % "2.10.0" % Test, // ApacheV2
```

Comment:
good that we are using the same Jackson version in all Lightbend products (Akka, Play, Lagom, ...)

Reply:
This is just in the non-published tests, in the testkit I didn't dare to go past 2.9.10.